### ISSUE TITLE
AttributeError: 'UpkieWheelsEnv' object has no attribute 'id'

### ISSUE THREAD

**Issue by Varun-GP** [tasts-robots/upkie_locomotion#22]
Thanks for this awesome project. I wanted to train the _ppo_balancer_ policy, however I am getting the following error, I have installed all the dependencies in _requirements_lock.txt_: 

```
Logging to /root/.cache/bazel/_bazel_root/b217c4be394e329054ec82691e211f34/execroot/upkie_locomotion/bazel-out/k8-fastbuild/bin/agents/ppo_balancer/train.runfiles/upkie_locomotion/agents/ppo_balancer/policies/dunce_env-v1_1
Traceback (most recent call last):
  File "/root/.cache/bazel/_bazel_root/b217c4be394e329054ec82691e211f34/execroot/upkie_locomotion/bazel-out/k8-fastbuild/bin/agents/ppo_balancer/train.runfiles/upkie_locomotion/agents/ppo_balancer/train.py", line 216, in <module>
    train_policy(agent_name, training_dir)
  File "/root/.cache/bazel/_bazel_root/b217c4be394e329054ec82691e211f34/execroot/upkie_locomotion/bazel-out/k8-fastbuild/bin/agents/ppo_balancer/train.runfiles/upkie_locomotion/agents/ppo_balancer/train.py", line 119, in train_policy
    policy.learn(
  File "/root/.cache/bazel/_bazel_root/b217c4be394e329054ec82691e211f34/execroot/upkie_locomotion/bazel-out/k8-fastbuild/bin/agents/ppo_balancer/train.runfiles/pip_upkie_locomotion_stable_baselines3/stable_baselines3/ppo/ppo.py", line 307, in learn
    return super().learn(
  File "/root/.cache/bazel/_bazel_root/b217c4be394e329054ec82691e211f34/execroot/upkie_locomotion/bazel-out/k8-fastbuild/bin/agents/ppo_balancer/train.runfiles/pip_upkie_locomotion_stable_baselines3/stable_baselines3/common/on_policy_algorithm.py", line 244, in learn
    callback.on_training_start(locals(), globals())
  File "/root/.cache/bazel/_bazel_root/b217c4be394e329054ec82691e211f34/execroot/upkie_locomotion/bazel-out/k8-fastbuild/bin/agents/ppo_balancer/train.runfiles/pip_upkie_locomotion_stable_baselines3/stable_baselines3/common/callbacks.py", line 70, in on_training_start
    self._on_training_start()
  File "/root/.cache/bazel/_bazel_root/b217c4be394e329054ec82691e211f34/execroot/upkie_locomotion/bazel-out/k8-fastbuild/bin/agents/ppo_balancer/train.runfiles/pip_upkie_locomotion_stable_baselines3/stable_baselines3/common/callbacks.py", line 194, in _on_training_start
    callback.on_training_start(self.locals, self.globals)
  File "/root/.cache/bazel/_bazel_root/b217c4be394e329054ec82691e211f34/execroot/upkie_locomotion/bazel-out/k8-fastbuild/bin/agents/ppo_balancer/train.runfiles/pip_upkie_locomotion_stable_baselines3/stable_baselines3/common/callbacks.py", line 70, in on_training_start
    self._on_training_start()
  File "/root/.cache/bazel/_bazel_root/b217c4be394e329054ec82691e211f34/execroot/upkie_locomotion/bazel-out/k8-fastbuild/bin/agents/ppo_balancer/train.runfiles/upkie_locomotion/agents/ppo_balancer/train.py", line 42, in _on_training_start
    self.env.id(),
  File "/root/.cache/bazel/_bazel_root/b217c4be394e329054ec82691e211f34/execroot/upkie_locomotion/bazel-out/k8-fastbuild/bin/agents/ppo_balancer/train.runfiles/pip_upkie_locomotion_gym/gym/core.py", line 238, in __getattr__
    return getattr(self.env, name)
AttributeError: 'UpkieWheelsEnv' object has no attribute 'id'
```

Also please upload any trained policies since the _policies_ directory is empty. This would be very much helpful for experimentation.

**Comment by stephane-caron** [comment]
Thank you for reporting this issue. It should be fixed with f8f05c35f98e959efe5dc52ded40b03e2f541c43.

**Comment by stephane-caron** [comment]
I've added the sample policy mentioned in [this discussion](https://github.com/tasts-robots/upkie_locomotion/discussions/15) to the repository. I'll re-upload a proper one (with training metadata: https://github.com/tasts-robots/upkie_locomotion/issues/23) but it should allow you to get started already.
